dashboard/test/ui/features/acquisition_products/pd/teacher_application.feature

```diff
@@ -185,22 +150,83 @@ Scenario: Teacher saves, re-opens, and submits an application
 
   # Section 3
   Then I wait until element "h3" contains text "Section 3: Professional Learning Program Requirements"
```

Comment:
Nit: It looks like you included the `Then I wait until element "h3" contains text “Section …”` inside the `I complete Section 1 of the teacher PD application` step. Could you do that for the others to be consistent?

Reply:
Good point, will do!